app/models/user.rb

```diff
@@ -24,8 +24,6 @@ class UserAuthenticationException < StandardError; end
   enum :role, {
     super_admin: "super_admin",
     organisation_admin: "organisation_admin",
-    editor: "editor",
-    trial: "trial",
```

Comment:
When I tested the code I tried to access the /users route and an exception was thrown because I hadn't run the rake task to convert trial users into standard users.

I'm not sure any code needs to change but maybe a reminder in slack telling people to run in their local environments would be handy?

It did make me wonder if it's safer to always keep old values of this enum in for safety. I think by this point we are confident that trial and editors should not exist and be treated as errors if they do? We also use strings instead of numbers so no users would suddenly change roles if the indexes change.

Reply:
Good catch @thomasiles! Yes we've gone through each environment and converted all trial and editor users to the standard role, so any trial or editors should be treated as errors from now on.

Hmm we've deleted the migration rake task...do you think we should retain this, or message that people should drop and reseed their local database?